tests/conftest.py

```diff
@@ -3386,6 +3386,49 @@ def data_context_simple_expectation_suite(tmp_path_factory):
     return ge.data_context.DataContext(context_path)
 
 
+@pytest.fixture
+def data_context_simple_expectation_suite_with_custom_pandas_dataset(tmp_path_factory):
+    """
+    This data_context is *manually* created to have the config we want, vs
+    created with DataContext.create()
+    """
+    project_path = str(tmp_path_factory.mktemp("data_context"))
+    context_path = os.path.join(project_path, "great_expectations")
+    asset_config_path = os.path.join(context_path, "expectations")
+    fixture_dir = file_relative_path(__file__, "./test_fixtures")
+    os.makedirs(
+        os.path.join(asset_config_path, "my_dag_node"),
+        exist_ok=True,
+    )
+    shutil.copy(
+        os.path.join(
+            fixture_dir, "great_expectations_basic_with_custom_pandas_dataset.yml"
+        ),
+        str(os.path.join(context_path, "great_expectations.yml")),
+    )
+    shutil.copy(
+        os.path.join(
+            fixture_dir,
+            "rendering_fixtures/expectations_suite_1.json",
+        ),
+        os.path.join(asset_config_path, "default.json"),
+    )
+    os.makedirs(os.path.join(context_path, "plugins"), exist_ok=True)
+    shutil.copy(
+        os.path.join(fixture_dir, "custom_pandas_dataset.py"),
+        str(os.path.join(context_path, "plugins", "custom_pandas_dataset.py")),
+    )
+    shutil.copy(
+        os.path.join(fixture_dir, "custom_sqlalchemy_dataset.py"),
+        str(os.path.join(context_path, "plugins", "custom_sqlalchemy_dataset.py")),
+    )
+    shutil.copy(
+        os.path.join(fixture_dir, "custom_sparkdf_dataset.py"),
+        str(os.path.join(context_path, "plugins", "custom_sparkdf_dataset.py")),
+    )
+    return ge.data_context.DataContext(context_path)
```

Comment:
Similar to an earlier comment: is this a fix for pre-`v013`?  Because `dataset.py` does not figure in `v013`.

Reply:
All the fixes in this PR are for pre-013 API.